package/cloudshell/iac/terraform/constants.py

```diff
@@ -42,7 +42,10 @@
 AZURE2G_MODEL = "Microsoft Azure Cloud Provider 2G"
 AWS2G_MODEL = "Amazon AWS Cloud Provider 2G"
 
-CLP_PROVIDER_MODELS = [AWS1G_MODEL, AWS2G_MODEL, AZURE1G_MODEL, AZURE2G_MODEL]
+GCP1G_MODEL = "Google Cloud Provider"
```

Comment:
The GCP shell is 2nd gen shell. We dont have 1st gen shell for GCP
https://community.quali.com/repos/4477/google-cloud-platform-cloud-provider